src/BulkManager/BulkAction/EditHandler.php

```diff
@@ -393,22 +395,24 @@ public function index()
      * and parses and saves each records data.
      *
      * @param array $data Sumitted form data
-     * @param Form  $form Form
+     * @param Form $form Form
+     * @return HTTPResponse
+     * @throws ValidationException
      */
     public function doSave($data, $form)
     {
-        $className = $this->gridField->list->dataClass;
+        $className = $this->gridField->getList()->dataClass;
```

Comment:
dataClass is a protected var. did you mean dataClass() method?

Reply:
There are a lot of accessing protected methods in these handlers. I'll try root out the rest of them. `gridField` is protected as well